### PR TITLE
Fix evidently visualizer on Colab

### PR DESCRIPTION
**Pull request by fa9r**
## Describe changes
I changed the evidently visualizer to correctly display its reports within Colab.
I also changed `in_google_colab()` of `environment.py` to support Colab CPU/TPU environments.

## Pre-requisites
Please ensure you have done the following:
- [x] I have read the **CONTRIBUTING.md** document.
- [x] If my change requires a change to docs, I have updated the documentation accordingly.
- [x] If I have added an integration, I have updated the [integrations](https://docs.zenml.io/features/integrations) table.
- [x] I have added tests to cover my changes.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Other (add details above)

### CODE REVIEW

**Review by schustmi — APPROVED** [review]

File: src/zenml/integrations/airflow/orchestrators/airflow_orchestrator.py [hunk 1]
```diff
@@ -96,7 +96,7 @@ def _translate_schedule(
             }
         return {
             "schedule_interval": "@once",
-           # set the a start time in the past and disable catchup so airflow runs the dag immediately
+            # set the a start time in the past and disable catchup so airflow runs the dag immediately
```
Comment (strickvl) [inline src/zenml/integrations/airflow/orchestrators/airflow_orchestrator.py:99]:
```suggestion
           # set the start time in the past and disable catchup so airflow runs the dag immediately
```

Wasn't sure of the purpose of this additional space here...

**Review by strickvl — APPROVED** [review]
Just one possible nit, otherwise looks good to me!

File: src/zenml/integrations/airflow/orchestrators/airflow_orchestrator.py (hunk 1 above)
Comment (fa9r) [reply src/zenml/integrations/airflow/orchestrators/airflow_orchestrator.py:99]: the format script automatically formatted it like this, seems like the line was wrongly indented before.

Comment (fa9r) [reply src/zenml/integrations/airflow/orchestrators/airflow_orchestrator.py:99]: just checked the file, indentation looks correct, so will mark this resolved

**Pull request merged**